library/src/components/connectors/destination/select-destination/SelectDestination.vue

```diff
@@ -125,12 +162,43 @@ export default {
       const regex = new RegExp(String.raw`${setting.schema.pattern}`)
       if (!setting.value) {
         return setting.schema.required ? 'Required' : true
+      } else if (setting.schema.type === "array") {
+        // NB: we don't bother recursively validating array elements in this hacky implementation
+        return this.validateArray(setting)
+      } else if (setting.schema.type === "number") {
+        return this.validateNumber(setting)
       } else if (regex && !regex.test(setting.value)) {
         return 'Invalid entry'
       } else {
         return true
       }
     },
+    validateArray(setting) {
+      const minItems = setting.schema.minItems
```

Comment:
Just realizing I made a change and this should be `min_items` (snake case). Will fix later today.